.eslintrc.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+	"env": {
+		"browser": true,
+		"commonjs": true,
+		"es6": true,
+		"node": true,
+		"jasmine": true
+	},
+	"plugins": [
+		"react"
+	],
+	"extends": "eslint:recommended",
+	"globals": {
+		"atom": "readonly"
+	},
+	"parserOptions": {
+		"ecmaVersion": 2018,
+		"ecmaFeatures": {
+			"jsx": true,
+		}
+	},
+	"rules": {
+		indent: ["error", "tab"],
+		semi: ["error", "always"],
+		quotes: ["error", "double"],
+		"jsx-quotes": ["error", "prefer-double"],
+
+		"react/jsx-uses-vars": 2,
```

Comment:
What means this "2" in the options?
Can't find the documentation for `react/jsx-uses-vars`.

Reply:
`2` = `"error"`

I'll change it.

Reply:
Ah yes, sorry my fault.